challenge-13/submissions/onomica/solution-template.go

```
@@ -0,0 +1,174 @@
+	}
+
+	// The table should have columns: id, name, price, quantity, category
+	query := `CREATE TABLE IF NOT EXISTS products (
+    	id INTEGER PRIMARY KEY AUTOINCREMENT,
+    	name TEXT NOT NULL,
+    	price REAL NOT NULL,
+		quantity INTEGER NOT NULL,
+		category TEXT NOT NULL
+	)`
+
+	_, err = db.Exec(query)
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)